.gitignore

```diff
@@ -1,3 +1,4 @@
 artifacts/
 coverage/
 node_modules/
+.venv/
```

Comment:
I think this isn't related to this project. Please use a global .gitignore instead.
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
You are right! I'll remove and look into that